File: 백준/Bronze/10869. 사칙연산/사칙연산.js
const input = require("fs").readFileSync(process.platform === "linux" ? "/dev/stdin" : "알고리즘/example.txt").toString().trim().split(" ").map(Number);

function calc(arr){
    let a = arr[0];
    let b = arr[1];
    let answer = [];
    answer.push(a+b);
    answer.push(a-b);
    answer.push(a*b);
    answer.push(parseInt((a/b)));
    answer.push(a%b);
    return answer;
}

function print(arr){
    for(let i = 0; i < arr.length; i++){
        console.log(arr[i]);
    }
    return 0;
}
let answer = calc(input);
print(answer);

File: 백준/Bronze/10869. 사칙연산/사칙연산.py
import sys
input = sys.stdin.readline

A, B = map(int, input().split(" "))

def plus(x,y):
    return x + y

def minus(x,y):
    return x - y

def multi(x,y):
    return x * y

def share(x,y):
    return x // y

def mod(x,y):
    return x % y

print(plus(A,B))
print(minus(A,B))
print(multi(A,B))
print(share(A,B))
print(mod(A,B))